src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1196,5 +1199,224 @@ internal static bool TryGetInt(FormulaValue value, out int outputValue)
             outputValue = (int)inputValue;
             return true;
         }
+
+#pragma warning disable SA1313 // Parameter '_' should begin with a lower-case letter
+
+        internal static FormulaValue Json(EvalVisitor runner, EvalVisitorContext _, IRContext irContext, FormulaValue[] args)
+        {
+            return new JsonProcessing(runner.TimeZoneInfo, irContext, args).Process();
+        }
+
+#pragma warning restore SA1313
+
+        internal class JsonProcessing

```

Comment:
>JsonProcessing [](http://example.com/codeflow?start=23&length=14)

Curious why are we implementing JSON de-serializer? 
Can't we reuse FormulaValueJson.FromJSON() ?
And move this to Fx.STJ?

Reply:
We have to follow the spec (PA Client reference), hence some custom code.
This code will be moved to Fx.Json later